packages/l10n/src/create-l10n-injector/create-l10n-injector.spec.tsx

```diff
@@ -1,11 +1,15 @@
-import React from 'react';
-import { shallow } from 'enzyme';
+import * as React from 'react';
+import { shallow, ShallowWrapper } from 'enzyme';
 import { reportErrorToSentry } from '@commercetools-frontend/sentry';
 import createL10NInjector from './create-l10n-injector';
 
 jest.mock('@commercetools-frontend/sentry');
 
-const mockData = {
+type Mock = Record<number, string>;
+
+type MockData = Record<string, Mock>;
```

Comment:
If you know the shape, you can simply do

```ts
type MockData = {
  en: Record<number, string>,
  de: Record<number, string>,
};
```

Reply:
hm, doesn't that break this line though?

```
const loadLocalesMock = jest.fn((locale, cb) => cb(null, mockData[locale]));
```

Reply:
Hmm how would it break it? 🤔